### DISCUSSION TITLE
Welcome to Screenshot_Application Discussions!

### DISCUSSION THREAD

**Discussion by KhushiJain2004** [KhushiJain2004/Screenshot_Application#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by KhushiJain2004** [comment]
Hi

**Comment by rajatkamboj06** [comment]
hii

**Comment by KhushiJain2004** [comment]
As you know we have to develop a Screenshot application ...
Do you have any ideas regarding it ??

**Comment by rajatkamboj06** [comment]
yeah
like the user should have the control to take a screenshot

**Comment by KhushiJain2004** [comment]
Are you reffering to time delay?

**Comment by rajatkamboj06** [comment]
yeah

**Comment by rajatkamboj06** [comment]
and how about burst screenshots instead of one?
that will be more precise...

**Comment by KhushiJain2004** [comment]
yeah we can but there will be many waste screenshots.
we cant switch to burst instead of one...

however we can make the user decide itself what kind of screenshot he/she wants...

**Comment by KhushiJain2004** [comment]
we can also ask the user about the name and destination of screenshot.

**Comment by rajatkamboj06** [comment]
yeah that will be more useful and precise

**Comment by rajatkamboj06** [comment]
how about the user can see all of its previous screenshots in a list

**Comment by KhushiJain2004** [comment]
no that will create a lot of chaos in GUI

**Comment by rajatkamboj06** [comment]
so we are using python na?

**Comment by KhushiJain2004** [comment]
yeah
was thinking of using tkinter module as it is easy to operate in that and it also gives us a descent GUI

**Comment by rajatkamboj06** [comment]
okk

**Comment by KhushiJain2004** [comment]
i have created a use case diagram

**Comment by KhushiJain2004** [comment]
![image](https://github.com/KhushiJain2004/Screenshot_Application/assets/129265117/467a0331-1cad-4284-adbd-2517d401139f)

**Comment by rajatkamboj06** [comment]
yeah this looks good....

**Comment by KhushiJain2004** [comment]
so lets finalize the functionalities we have till now

**Comment by rajatkamboj06** [comment]
okk so....

1. time delay with single screenshot
2.multiple screenshot a the same time
3.user friendly loacion and name service
4.window minimization when any of the option is choosen
5.we are using tkinter module

**Comment by KhushiJain2004** [comment]
we can create a tab like GUI that will go handy with these functions

**Comment by rajatkamboj06** [comment]
how about a about page that can halp any new user about the project.

**Comment by KhushiJain2004** [comment]
yesss

**Comment by KhushiJain2004** [comment]
![WhatsApp Image 2023-09-26 at 00 32 59](https://github.com/KhushiJain2004/Screenshot_Application/assets/129265117/591e8825-92cb-4bd4-931b-cfdfd54ca7a0)

**Comment by KhushiJain2004** [comment]
this is the GUI overview

**Comment by rajatkamboj06** [comment]
this looks great

**Comment by KhushiJain2004** [comment]
lets recall the functionalities in case we forgot any
so
time delay with single screenshot
2.multiple screenshot a the same time
3.user friendly loacion and name service
4.window minimization when any of the option is choosen
5.we are using tkinter module
and a additional option to exit the code in case user dont want that

**Comment by KhushiJain2004** [comment]
hii..
 i have uploaded the basic layout for the application window 
do check it out

**Comment by rajatkamboj06** [comment]
okkk i wll change my part too

**Comment by KhushiJain2004** [comment]
As discussed , i'll be working on  the layout and functions for:
a)single screenshot tab and 
b)multiple screenshot tab ..

and you will be responsible for the about tab in the window  and the menu bar 
please update when done ..

**Comment by rajatkamboj06** [comment]
I have updated the menu bar ...

**Comment by KhushiJain2004** [comment]
i have updated both .. single and multiple screenshots tabs
